CustomCraftSML/Serialization/Entries/CustomFood.cs

```diff
@@ -0,0 +1,414 @@
+            set => emForceAtStart.Value = value;
+        }
+        protected readonly EmYesNo emForceAtStart;
+        protected const string ForceUnlockKey = "ForceUnlockAtStart";
+
+        public IList<EmIngredient> Ingredients => emIngredient.Values;
+        protected readonly EmPropertyCollectionList<EmIngredient> emIngredient;
+        protected List<Ingredient> IngredientsItems { get; } = new List<Ingredient>();
+        protected const string IngredientsKey = "Ingredients";
+
+        public IList<string> LinkedItemIDs => emLinked.Values;
+        protected readonly EmPropertyList<string> emLinked;
+        protected List<TechType> LinkedItems { get; } = new List<TechType>();
+        protected const string LinkedKey = "LinkedItemIDs";
+
+        public IList<string> Unlocks => emUnlocks.Values;
+        protected readonly EmPropertyList<string> emUnlocks;
+        protected List<TechType> UnlocksItems { get; } = new List<TechType>();
+        protected const string UnlocksKey = "Unlocks";
+
+        public IList<string> UnlockedBy => emUnlockedby.Values;
+
+        public TechType UnlockedBy_
+        {
+            get
+            {
+                if (!ForceUnlockAtStart && UnlockedBy.Count > 0)
+                {
+                    TechType item = GetTechType(UnlockedBy[0]);
+                    return item;
+                }
+                else
+                {
+                    return TechType.None;
+                }
+            }
+        }
+        protected readonly EmPropertyList<string> emUnlockedby;
+        protected List<TechType> UnlockedByItems { get; } = new List<TechType>();
+        protected const string UnlockedbyKey = "UnlockedBy";
+
+        public string DisplayName
+        {
+            get => emDisplayname.Value;
+            set => emDisplayname.Value = value;
+        }
+        protected readonly EmProperty<string> emDisplayname;
+        protected const string DisplayNameKey = "DisplayName";
+
+        public string Tooltip
+        {
+            get => emTooltip.Value;
+            set => emTooltip.Value = value;
+        }
+        protected readonly EmProperty<string> emTooltip;
+        protected const string TooltipKey = "Tooltip";
+
+        public TechType SpriteItemID
+        {
+            get => emSpriteID.Value;
+            set => emSpriteID.Value = value;
+        }
+        protected readonly EmProperty<TechType> emSpriteID;
+        protected const string SpriteKey = "SpriteItemID";
+
+        public string Path
+        {
+            get => emPath.Value;
+            set => emPath.Value = value;
+        }
+        protected readonly EmProperty<string> emPath;
+        protected const string PathKey = "Path";
+
+        public TechGroup PdaGroup
+        {
+            get => emPdagroup.Value;
+            set => emPdagroup.Value = value;
+        }
+        protected readonly EmProperty<TechGroup> emPdagroup;
+        protected const string PdagroupKey = "PdaGroup";
```

Comment:
`PdaGroup` is _always_ going to be `TechGroup.Survival`.  
We can remove all usages of this property and simply set it directly in the Craftable class.